popSimManu.tex

```diff
@@ -337,21 +337,27 @@ \subsection*{The \stdpopsim Catalog}
 Gutenkunst model which does not include Asian populations, but more explicitly models
 recent rapid human population growth, the \cite{browning2018ancestry} admixture model
 for American populations which specifies ancestral African, European, and Asian population
-components, and the three-population out-of-Africa model from \cite{ragsdale2019models}
-which includes archaic admixture.
+components, a three-population out-of-Africa model from \cite{ragsdale2019models}
+which includes archaic admixture, a complex model of ancient eurasian admixture from \cite{kamm2019efficiently},
+and a synthetic model of oscillating population size from \cite{schiffels2014inferring}.
 Together these models
 contain features believed to have widespread impacts in real data (e.g. bottlenecks, population growth,
 admixture) and are therefore highly pertinent in the context of method development.
 
 For non-human genomes we have have implemented two demographic histories for
-\emph{Drosophila melanogaster} and one each from \emph{A. thaliana} and \emph{E. coli}.
+\emph{Drosophila melanogaster}, three from \emph{A. thaliana}, an one each from \emph{P. abelii} and \emph{E. coli}.
```

Comment:
Is there one E. coli model? it's not in the Catalog.

Reply:
It's got the null single population model that all species have